CHANGELOG.md

```diff
@@ -131,6 +131,8 @@ and this project adheres to
   - [#4244](https://github.com/bpftrace/bpftrace/pull/4244)
 - Report expanded probe name when trying to attach
   - [#4353](https://github.com/bpftrace/bpftrace/pull/4353)
+- Fix builds on PowerPC
```

Comment:
@viktormalik Is this something broken on the current version or is it just in trunk after @amscanne 's refactoring